env/env.go

```diff
@@ -1,25 +1,40 @@
 package env
 
 import (
-	"github.com/EVODelavega/goq/aws"
+	"os"
+
+	goqaws "github.com/EVODelavega/goq/aws"
 
 	"github.com/caarlos0/env"
 )
 
 // Get parses the variable environment and creates the config object
-func Get() (*aws.Config, error) {
+func Get() (*goqaws.Config, error) {
 
-	credentials := &aws.Credentials{}
+	credentials := &goqaws.Credentials{}
 	if err := env.Parse(credentials); err != nil {
 		return nil, err
 	}
 
-	config := &aws.Config{}
+	config := &goqaws.Config{}
 	if err := env.Parse(config); err != nil {
 		return nil, err
 	}
 
+	// Parse manually properties, that are presented as pointers
+	config.QueueUrl = getStringPtr(os.Getenv("AWS_QUEUE_URL"))
```

Comment:
the env package has customer parser support. Also,  the queue url or topic ARN might not be set, in which case `os.LookupEnv` might be the better function to use

Reply:
Good point